Managers/LuaMan.cpp

```diff
@@ -437,6 +437,12 @@ int LuaMan::Create()
 	lua_pushliteral(m_pMasterState, LUA_LOADLIBNAME);
 	lua_call(m_pMasterState, 1, 0);
 
+	lua_pushcfunction(m_pMasterState, luaopen_jit);
```

Comment:
I recall the luabind docs suggesting not to use the individual pushcfunction stuff and just using full loading, but that not working cause we need to get rid of io. It's ultimately fine as is, but if you wanna try and make it cleaner, [this thread](https://stackoverflow.com/questions/10108933/how-to-remove-particular-functions-from-os-library-without-editing-the-lua-heade) might help.

Reply:
Done, #173